custom_components/plugwise_usb/coordinator.py

```
@@ -37,6 +37,9 @@ def __init__(
     ) -> None:
         """Initialize Plugwise USB data update coordinator."""
         self.node = node
+        self.subscribed_nodefeatures: list[NodeFeature] = []
+        self._subscribe_to_feature_fn = self.node.subscribe_to_feature_update
+        self.unsubscribe_push_events: Callable[[], None] | None = None
```

Ruff (0.11.9)

Undefined name `Callable`

(F821)